### PR TITLE
Feature/user can favorite or frown a card

### PR DESCRIPTION
**Pull request by auggernaut**

### CODE REVIEW

**Comment by jacksenechal** [comment]
Two problems I see when testing the functionality:

1. The frown count is not shown until you frown a card. Should behave like favorites.
2. The frown count is not reset when you go to the next card. Since we reuse one CardView and just load new data into it, all data needs to be cleared before the next card is loaded. I suspect this is related to #1.

File: src/app/stores/helpers/ParseBackend.coffee [hunk 1]
```diff
@@ -158,6 +160,31 @@ class ParseBackend
...
+      .then (result) =>
+        if result?
+          frowns = result.get 'frowns'
+          if frowns?
+            frowns += 1
+          else
+            frowns = 1
+          result.set 'frowns', frowns
+          result.save()
+
```
Comment (auggernaut) [inline src/app/stores/helpers/ParseBackend.coffee]: Found a section on Counters in the parse docs. Looks like this code can be shortened to:
result.increment 'frowns'
result.save()

File: src/app/stores/helpers/ParseBackend.coffee [hunk 2]
```diff
@@ -191,6 +218,39 @@ class ParseBackend
...
+      .then (result) =>
+        if result?
+          frowns = result.get 'frowns'
+          if frowns?
+            frowns -= 1
+          else
+            frowns = 0
+          result.set 'frowns', frowns
+          result.save()
+
```
Comment (auggernaut) [inline src/app/stores/helpers/ParseBackend.coffee]: Similar to above:
result.decrement 'frowns'
result.save()

**Comment by justinriggio** [comment]
I'm going to do this to saveFavorites and deleteFavorites too.

**Comment by justinriggio** [comment]
@auggernaut its ready for review now.

**Pull request merged**